vendor/bat-native-ads/src/bat/ads/internal/ads/serving/eligible_ads/exclusion_rules/subdivision_targeting_exclusion_rule_ca_unittest.cc

```diff
@@ -0,0 +1,300 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ads/serving/eligible_ads/exclusion_rules/subdivision_targeting_exclusion_rule.h"
+
+#include <memory>
+
+#include "bat/ads/internal/base/unittest/unittest_base.h"
+#include "bat/ads/internal/base/unittest/unittest_mock_util.h"
+#include "bat/ads/internal/geographic/subdivision/subdivision_targeting.h"
+#include "bat/ads/pref_names.h"
+#include "net/http/http_status_code.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+
+namespace {
+constexpr char kCreativeSetId[] = "654f10df-fbc4-4a92-8d43-2edf73734a60";
+}  // namespace
+
+class BatAdsSubdivisionTargetingExclusionRuleCATest : public UnitTestBase {
```

Comment:
Tests for different regions are identical. So rather than having both _us and _ca tests could we change to use parameterized tests?